sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -1076,7 +1130,8 @@
                       },
                       "tableView": true,
                       "validate": {
-                        "required": true
+                        "required": true,
```

Comment:
It does not need to be persistent unless I am missing something. 
![image](https://github.com/user-attachments/assets/74c163ae-5ff0-43d4-a42f-86eb70934f4a)